src/Altinn.App.Core/Internal/Process/EventHandlers/ProcessTask/EndTaskEventHandler.cs

```diff
@@ -0,0 +1,71 @@
+﻿using Altinn.App.Core.Features;
+using Altinn.App.Core.Internal.Process.ProcessTasks;
+using Altinn.App.Core.Internal.Process.ServiceTasks;
+using Altinn.Platform.Storage.Interface.Models;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Altinn.App.Core.Internal.Process.EventHandlers.ProcessTask
+{
+    /// <summary>
+    /// This event handler is responsible for handling the end event for a process task.
+    /// </summary>
+    public class EndTaskEventHandler : IEndTaskEventHandler
+    {
+        private readonly IProcessTaskDataLocker _processTaskDataLocker;
+        private readonly IProcessTaskFinalizer _processTaskFinisher;
+        private readonly IServiceTask _pdfServiceTask;
+        private readonly IServiceTask _eformidlingServiceTask;
+        private readonly IEnumerable<IProcessTaskEnd> _processTaskEnds;
+
+        /// <summary>
+        /// This event handler is responsible for handling the end event for a process task.
+        /// </summary>
+        public EndTaskEventHandler(
```

Comment:
This code used to be in DefaultTasksEvents. Common "end task" logic for all process tasks.